dockstore-client/src/main/java/io/dockstore/client/cli/nested/WesLauncher.java

```diff
@@ -329,8 +333,8 @@ public static String createWorkflowTypeVersion(String workflowType) {
      * @param runId The ID of the launched workflow. This ID is provided from the WES server.
      */
     public static void wesCommandSuggestions(String runId) {
-        out("To view the workflow run status, execute: ");
-        out(MessageFormat.format("\tdockstore workflow wes status --id {0}", runId));
-        out(MessageFormat.format("\tdockstore workflow wes logs --id {0}", runId));
+        out("To view the workflow run " + STATUS + ", execute: ");
+        out(MessageFormat.format("\tdockstore workflow " + WES + " " + STATUS + " " + ID + " {0}", runId));
+        out(MessageFormat.format("\tdockstore workflow " + WES + " " + LOGS + " " + ID + " {0}", runId));
```

Comment:
You can take advantage of `MessageFormat.format` here so you don't have to concat the constants

Reply:
Fixed